tests/test_detectors.py

```diff
@@ -196,6 +196,14 @@ def test_autoencoder_detector(data_series):
     # assert all(i in expected_anomalies_indices for i in anomalies_indices)
 
 
+def test_autoencoderlstm_detector(data_series):
+    data_with_anomalies, expected_anomalies_indices, normal_data = data_series
+    detector = AutoEncoderLSTM()
+    detector.fit(data_with_anomalies)
+    anomalies = detector.detect(data_with_anomalies)
+    anomalies_indices = np.array(np.where(anomalies)).flatten()
+
```

Comment:
I am a bit confused about what is being tested, I was expecting to see an assert to compare between expected and detected anomalies?

Reply:
Yeah. I did not create a full test as I was unsure what to expect from this very small artificial test dataset and the LSTM output. We should create a better test for this detector.

Reply:
Perhaps some random numbers with high spikes in a few locations as test data?

Reply:
The detector finds a lot of anomalies right now in the artificial test set. I think we need more time to work out a good case. I will create an issue that the autoencoders should be tested properly in order to close this PR.